### PR TITLE
Operators - support multi-operators install/uninstall

### PR DESCRIPTION
**Pull request by rnetser**
Requires https://github.com/RedHatQE/openshift-python-utilities/pull/214

### CODE REVIEW

File: app/operator_cmds/commands.py [hunk 1]
```diff
@@ -10,11 +12,76 @@ def _client(ctx):
...
+    """,
+    required=True,
+    multiple=True,
+)
+@click.option(
+    "-p",
+    "--parallel",
+    help="Run operator install/uninstall in parallel",
+    default="false",
+    type=click.Choice(["true", "false"]),
```
Comment (myakove) [inline app/operator_cmds/commands.py:78]: Should be bool, not Choice

File: app/operator_cmds/commands.py [hunk 2]
```diff
@@ -26,71 +93,62 @@ def _client(ctx):
...
     ctx.obj["timeout"] = timeout
     ctx.obj["kubeconfig"] = kubeconfig
-    ctx.obj["namespace"] = namespace
+    ctx.obj["parallel"] = ast.literal_eval(parallel.capitalize())
     if debug:
         os.environ["OCM_PYTHON_WRAPPER_LOG_LEVEL"] = "DEBUG"
         os.environ["OPENSHIFT_PYTHON_WRAPPER_LOG_LEVEL"] = "DEBUG"
 
+    operators_dict = {}
+    for _operator in [__operator for __operator in operators if __operator]:
```
Comment (myakove) [inline app/operator_cmds/commands.py:111]: `if __operator` why this is needed?

File: app/operator_cmds/commands.py [hunk 3]
```diff
@@ -26,71 +93,62 @@ def _client(ctx):
...
+        operator_parameters = {}
+        operator_and_params = _operator.split("|")
+        operator_name = operator_and_params[0]
+
+        if len(operator_and_params) > 1:
+            parameters = operator_and_params[-1].split(",")
+            parameters = [_param.strip() for _param in parameters]
+            for parameter in parameters:
```
Comment (myakove) [inline app/operator_cmds/commands.py:119]: duplicate code?

**Review by myakove — CHANGES_REQUESTED** [review]
please update the readme

File: app/operator_cmds/commands.py (hunk 1 above)
Comment (rnetser) [reply app/operator_cmds/commands.py]: done

File: app/operator_cmds/commands.py (hunk 3 above)
Comment (rnetser) [reply app/operator_cmds/commands.py]: done

**Review by myakove — DISMISSED** [review]

**Pull request merged**